sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/Beta.java

```diff
@@ -55,5 +55,6 @@
         V4_9_0,
         /** v4.11.0 */
         V4_11_0,
+        WHATEVER_NEW_VERSION
```

Comment:
TODO - change this to whatever is the version at the time of merge before actually merging!